### ISSUE TITLE
Question, is there a way to get Extent Reports to display specflow tables like Specflow +?

### ISSUE THREAD

**Issue by Moshex** [extent-framework/extentreports-csharp#117]
Right now extent reports just displays the step and not the table to the step if there is one. is there a way to have extent reports to display the table to a step like in the screenshot below?

![Untitled](https://user-images.githubusercontent.com/6087936/105557542-bd5b6f00-5cc9-11eb-80d5-34f6183b8d25.png)

**Comment by anshooarora** [comment]
You can try with: https://www.extentreports.com/docs/versions/5/net/index.html#markup-table

This version is currently in alpha and I am trying to fix a few dependencies as well which broke due to compatibility issues with Standard/Core Tfm.

Nuget: https://www.nuget.org/packages/ExtentReports/5.0.0-alpha5

This build is nearly identical to the Java implementation, with the exception of a few features which will be available in the coming few weeks/months.

**Comment by MaheshGooner** [comment]
@anshooarora does this version with klov reporting? it doesnt seem to have `ExtentKlovReporter` implementation so wondering how can i get that to work.

**Comment by Moshex** [comment]
@anshooarora do you know if Extent reports version 5.x.x will support .Net Framework 4.8?

**Comment by anshooarora** [comment]
@MaheshGooner yes, the Klov client will be added soon.  5.x code is completely new so only prioritizing  the core API since the Klov client will be a near identical port from 4.x. 

@Moshex The latest alpha-5 targets .Net Standard 2.0 which supports .NET Framework 4.6.1+.  

<img width="899" alt="Screen Shot 2021-03-05 at 11 27 07 AM" src="https://user-images.githubusercontent.com/4107384/110164276-fb0de600-7da5-11eb-94c3-2c62450281c6.png">

**Comment by anshooarora** [comment]
@MaheshGooner KlovReporter is now available:  https://www.nuget.org/packages/klov-reporter/5.0.0-alpha5

This is a standalone dependency in version 5 (similar to Java).

**Comment by MaheshGooner** [comment]
That's great @anshooarora. I will let you know the feedback 👍

**Comment by Moshex** [comment]
@anshooarora Thank you for all your help! would you happen to have a code example of how to get the spec flow table to display on the report?

**Comment by anshooarora** [comment]
@Moshex  I am not sure how Specflow creates the table for examples,  will look into this.

**Comment by anshooarora** [comment]
fyi - entire codebase now on develop-5.0.0 branch:  https://github.com/extent-framework/extentreports-csharp/tree/develop-5.0.0

**Comment by Moshex** [comment]
@anshooarora The link you posted is broken, thank you for looking into this!
